task/net/src/main.rs

```diff
@@ -85,10 +85,15 @@ const TX_RING_SZ: usize = 4;
 const RX_RING_SZ: usize = 4;
 
 /// Notification mask for our IRQ; must match configuration in app.toml.
-const ETH_IRQ: u32 = 1;
+const ETH_IRQ: u32 = 1 << 0;
 
 /// Notification mask for optional periodic logging
-const WAKE_IRQ: u32 = 2;
+const WAKE_IRQ: u32 = 1 << 1;
+
+// NOTE: bit 3 is used in the BSPs!
```

Comment:
What about bit 2? Is there a nicer way to double check this since it looks like the TOML only shows the two IRQs?

Reply:
I'm honestly not sure what's going on here – it dates back to @cbiffle 's original commit.

Consolidating to using bits 0,1,2 seems to work fine, so let's do that!